### PR TITLE
Fix/feedbackstatus

### PR DESCRIPTION
**Pull request by rlin0**
### Summary
- Change status to resolved/unresolved instead of read/unread 
- Filter by resolved status in summary view
- Also send update email to group members
- minor style changes in summary view

### Test Plan
- Tested posting feedback and adding messages

### CODE REVIEW

File: donut/modules/feedback/helpers.py [hunk 1]
```diff
@@ -192,31 +193,17 @@ def get_status(group, complaint_id):
...
-    returns False if complaint_id is invalid
+    Sets the status of this complaint to resolved/unresolved
     '''
-    if get_status(group, complaint_id) is None:
-        return False
-    query = "UPDATE complaint_info SET status = 'new_msg' WHERE complaint_id = %s"
+    query = "UPDATE complaint_info SET resolved=%s WHERE complaint_id = %s"
     with flask.g.pymysql_db.cursor() as cursor:
-        cursor.execute(query, complaint_id)
+        print(status)
```
Comment (calebsander) [inline donut/modules/feedback/helpers.py]: `print` should be removed

File: donut/modules/feedback/helpers.py [hunk 2]
```diff
@@ -102,20 +99,24 @@ def add_msg(group, complaint_id, message, poster, notification=True):
     "(anonymous)"
     If complaint_id is invalid, returns False
     '''
-    if not get_subject(group, complaint_id): return False
+    subj = get_subject(group, complaint_id)
```
Comment (calebsander) [inline donut/modules/feedback/helpers.py]: Nit, but generally clearer to use full words as variable names, e.g. `subject`

File: donut/modules/feedback/templates/summary.html [hunk 3]
```diff
@@ -3,17 +3,21 @@
 <div class="container theme-showcase" role="main">
 <div class="jumbotron" style="margin: 5px;">
 <h2>{{group}} Summary</h2>
+<form method="POST" action="{{ url_for('feedback.feedback_view_summary', group=group) }}">
+                            <label for="show">Filter by Unresolved</label>
```
Comment (calebsander) [inline donut/modules/feedback/templates/summary.html]: Maybe "show only unresolved feedbacks"?

File: donut/modules/groups/utils/insert_permissions.py [hunk 4]
```diff
@@ -65,14 +65,14 @@
...
-                arc_permissions.TOGGLE_READ,
+                arc_permissions.TOGGLE_RESOLVED,
                 arc_permissions.ADD_REMOVE_EMAIL,
                 arc_permissions.VIEW_EMAILS,
                 voting_permissions.SURVEYS],
         # View Arcfeedback summary, Edit Arcfeedback, View Arcfeedback emails,
         # Surveys
         False: [arc_permissions.SUMMARY,
-                arc_permissions.TOGGLE_READ,
+                arc_permissions.TOGGLE_RESOLVED,
```
Comment (calebsander) [inline donut/modules/groups/utils/insert_permissions.py:75]: Did you change the name of these permissions in the DB? They should also probably be changed on the wiki page: https://github.com/ASCIT/donut/wiki/Managing-the-production-server#initial-configuration

File: donut/modules/feedback/helpers.py [hunk 5]
```diff
@@ -192,31 +193,17 @@ def get_status(group, complaint_id):
...
-    if get_status(group, complaint_id) is None:
-        return False
-    query = "UPDATE complaint_info SET status = 'read' WHERE complaint_id = %s"
-    with flask.g.pymysql_db.cursor() as cursor:
-        cursor.execute(query, complaint_id)
+    return res['resolved'] if res else None
 
 
-def mark_unread(group, complaint_id):
+def set_status(group, complaint_id, status):
```
Comment (calebsander) [inline donut/modules/feedback/helpers.py]: `set_resolved()`?

**Review by calebsander — CHANGES_REQUESTED** [review]

File: donut/modules/feedback/helpers.py (hunk 5 above)
Comment (rlin0) [reply donut/modules/feedback/helpers.py]: because `set_status()` to match `get_status()`

**Review by calebsander — APPROVED** [review]

**Pull request merged**